cfme/tests/pod/test_appliance_crud.py

```diff
@@ -324,3 +342,59 @@ def test_aws_smartstate_pod():
         initialEstimate: None
     """
     pass
+
+
+def test_pod_appliance_db_backup_restore(temp_pod_appliance, provider, setup_provider,
+                                         template_folder):
+    """
+    This test does the following:
+      - adds openshift provider where openshift based appliance is situated
+      - creates pvc and db backup in it
+      - stops appliance
+      - restores db from snapshot
+      - starts appliance and finds that appliance in CloudForms
+    """
+    template_folder = template_folder
+    appliance = temp_pod_appliance
+    assert provider.mgmt.is_vm_running(appliance.project)
+
+    # add backup pvc
+    output = read_host_file(appliance, os.path.join(template_folder, 'cfme-backup-pvc.yaml'))
```

Comment:
Consider some assertion that cfme-backup-pvc.yaml exists before trying to read it into the appliance, to skip/fail cleanly.

Reply:
Could probably be up in the fixture

Reply:
HA ignore me this is in read_host_file, duh.